src/Aspire.Dashboard/Utils/DashboardUrls.cs

```diff
@@ -136,4 +136,12 @@ public static string LoginUrl(string? returnUrl = null, string? token = null)
 
         return url;
     }
+
+    public static string SetLanguageUrl(string language, string redirectUrl)
+    {
+        var languageEscaped = Uri.EscapeDataString(language);
+        var redirectUrlEscaped = Uri.EscapeDataString(redirectUrl);
+
+        return $"/api/set-language?language={languageEscaped}&redirectUrl={redirectUrlEscaped}";
```

Comment:
Is there a reason you chose `EscapeDataString` over `QueryHelpers.AddQueryString` to build up the URL? AddQueryString seems like it's the best tool for the job.

Reply:
No reason, changed it.